### PR TITLE
various material & ore inhands

### PR DESCRIPTION
**Pull request by muburu**
## About the PR
adds inhands for a few materials that were missing them, and inhands for ores. i'll be making more of these PRs in the future for more inhands.

list of stuff i inhanded

- cardboard
- bananium
- corgi pelt
- cloth
- cotton
- durathread
- silk
- rods
- pyrotton [with an o this time]
- bones
- goliath hide
- bananium ore
- coal
- diamond ore
- gold ore
- steel ore
- plasma ore
- salt
- silver ore
- spacequartz
- uranium ore

## Why / Balance
waiter waiter more inhands please!!
resolves a chunk of https://github.com/space-wizards/space-station-14/issues/33386

## Technical details
png additions meta stuff and yml blah blah blah

## Media
**various materials!!**
matinhands

**ore!!**
oreinhands

- suggestions to improve these sprites are welcome
- if you think i forgot anything feel free to complain (only stuff in the "materials" folder counts)
- whiter sprites might be hard to see if you use github light

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

no changelog for you

### CODE REVIEW

**Review by Flareguy — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Materials/materials.yml [hunk 1]
```diff
@@ -479,6 +486,8 @@
     tags:
       - ClothMade
       - RawMaterial
+  - type: Item
+    heldPrefix: pyrotten
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Materials/materials.yml:490]:
```suggestion
    heldPrefix: pyrotton
```

File: Resources/Textures/Objects/Materials/materials.rsi/meta.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24 , bear pelt remade by Alekshhh, wood sprite modified by MisterMecky, wood_2 and wood_3 made by MisterMecky based on wood sprite, cardboard sprites made by MisterMecky, bananium, bananium_1 and peel made by brainfood1183 (github) for ss14. Pyrotton sprites are drawn by Ubaser, using the cotton material sprites as a base.",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24 , bear pelt remade by Alekshhh, wood sprite modified by MisterMecky, wood_2 and wood_3 made by MisterMecky based on wood sprite, cardboard sprites made by MisterMecky, bananium, bananium_1 and peel made by brainfood1183 (github) for ss14. Pyrotton sprites are drawn by Ubaser, using the cotton material sprites as a base. Bananium, cardboard, corgihide, cloth, cotton, pyrotten, bones and durathread inhands by mubururu_ (github).",
```
Comment (slarticodefast) [inline Resources/Textures/Objects/Materials/materials.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24 , bear pelt remade by Alekshhh, wood sprite modified by MisterMecky, wood_2 and wood_3 made by MisterMecky based on wood sprite, cardboard sprites made by MisterMecky, bananium, bananium_1 and peel made by brainfood1183 (github) for ss14. Pyrotton sprites are drawn by Ubaser, using the cotton material sprites as a base. Bananium, cardboard, corgihide, cloth, cotton, pyrotton, bones and durathread inhands by mubururu_ (github).",
```

File: Resources/Textures/Objects/Materials/materials.rsi/meta.json [hunk 3]
```diff
@@ -69,6 +109,14 @@
     {
       "name": "pyrotton"
     },
+    {
+      "name": "pyrotten-inhand-left",
```
Comment (slarticodefast) [inline Resources/Textures/Objects/Materials/materials.rsi/meta.json:113]:
```suggestion
      "name": "pyrotton-inhand-left",
```

File: Resources/Textures/Objects/Materials/materials.rsi/meta.json [hunk 4]
```diff
@@ -69,6 +109,14 @@
...
+      "directions": 4
+    },
+    {
+      "name": "pyrotten-inhand-right",
```
Comment (slarticodefast) [inline Resources/Textures/Objects/Materials/materials.rsi/meta.json:117]:
```suggestion
      "name": "pyrotton-inhand-right",
```
rename the pngs as well

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Materials/materials.yml (hunk 1 above)
Comment (muburu) [reply Resources/Prototypes/Entities/Objects/Materials/materials.yml:490]: wait what i thought it was an e not an o lol

**Comment by muburu** [comment]
minor spelling error fixed!!!
the pyrotton is no longer rotten

**Comment by slarticodefast** [comment]
o7

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**